docs/userguide/networking/default_network/build-bridges.md

```diff
@@ -0,0 +1,73 @@
+<!--[metadata]>
++++
+title = "Build your own bridge"
+description = "Learn how to build your own bridge interface"
+keywords = ["docker, bridge, docker0, network"]
+[menu.main]
+parent = "smn_networking_def"
++++
+<![end-metadata]-->
+
+# Build your own bridge
+
+The information in this section explains building your own bridge to replace within the Docker default bridge. This is a `bridge` network named
+`bridge` created automatically when you install Docker.  
+
+**Note**: The [Docker networks feature](../dockernetworks.md) allows you to create user-defined networks in addition to the default bridge network.
+
+
+You can set up your own bridge before starting Docker and use `-b BRIDGE` or `--bridge=BRIDGE` to tell Docker to use your bridge instead.  If you already have Docker up and running with its default `docker0` still configured, you will probably want to begin by stopping the service and removing the interface:
+
+```
+# Stopping Docker and removing docker0
+
+$ sudo service docker stop
+$ sudo ip link set dev docker0 down
+$ sudo brctl delbr docker0
+$ sudo iptables -t nat -F POSTROUTING
+```
+
+Then, before starting the Docker service, create your own bridge and give it
+whatever configuration you want.  Here we will create a simple enough bridge
+that we really could just have used the options in the previous section to
+customize `docker0`, but it will be enough to illustrate the technique.
+
+```
+# Create our own bridge
+
+$ sudo brctl addbr bridge0
+$ sudo ip addr add 192.168.5.1/24 dev bridge0
+$ sudo ip link set dev bridge0 up
+
+# Confirming that our bridge is up and running
```

Comment:
Like the other files; I'd break up this code block, and put the comments outside code